README.md

```diff
@@ -47,64 +51,38 @@ java -Dconfig.file=wdls/shifter.conf -jar /full/path/to/cromwell-XX.jar run -i i
 
 ## Docker images
 
-- `microbiomedata/meta_t:latest`. 
-`Dockerfile` can be found in `Docker/metatranscriptomics/` directory. 
-- `microbiomedata/bbtools:38.94`
-- `scanon/nmdc-meta:v0.0.1`
-- `bfoster1/img-omics:0.1.7`
-- `scanon/im-trnascan:v0.0.1`
-- `scanon/im-last:v0.0.1`
-- `scanon/im-hmmsearch:v0.0.0`
+- `microbiomedata/meta_t:latest`
```

Comment:
We should use a tagged image here, not latest

Reply:
I didn't see any tags on the docker repo, we'll have to make a new one https://hub.docker.com/r/microbiomedata/meta_t/tags

Reply:
@chienchi  i think alicia meant to mention you here. I've tried to build the docker image from the commands at the top of the dockerfile, but I run into errors with `continuumio/miniconda3:4.8.2`

```
ERROR: failed to solve: process "/bin/sh -c apt-get update -y     && apt-get install -y build-essential unzip wget curl gawk locales    && apt-get clean" did not complete successfully: exit code: 100
```

I was able to build an image by bumping the version to `continuumio/miniconda3:4.10.3` but I don't have push permissions to the docker repo
https://github.com/microbiomedata/metaT/blob/workflows/docker/Dockerfile
@aclum it seems like you have push permissions as well, if you have the time to make/push the image

Reply:
I tried to build the image but it seems `conda` takes long time.  @kaijli I added the meta_t docker repo into developer team on docker hub which you are member of. Please try to push the updated image. However, question here is "do we still need this image after update to JGI workflow?"

Reply:
@chienchi I was trying to build a new image with just these python packages for the json tasks, but kept running into issues with my dockerfile

```
channels:
  - conda-forge
  - bioconda
dependencies:
  - pandas=2.1.4
  - gffutils=0.13
  - python=3.11.0
  ```
  python version can be lower, it runs on 3.7.12

Reply:
update to close thread: was able to make tagged releases for this new workflow, will need to make new images in future, tagged images have issue with old workflow somehow